cluster-up/cluster/okd-4.3/provider.sh

```diff
@@ -41,7 +41,7 @@ function up() {
     # Copy k8s config and kubectl
     cluster_container_id=$(docker ps -f "name=$provider_prefix-cluster" --format "{{.ID}}")
 
-    _install_from_cluster $cluster_container_id /usr/bin/oc 0755 .kubectl
+    _install_from_cluster $cluster_container_id /usr/local/bin/oc 0755 .kubectl
```

Comment:
I don't think this is part of what you are changing, it probably got generated when you ran make deps-update. Its a bug in kubevirtci because the oc executable in in /usr/bin in the okd 4.3 provider.

Reply:
Yes, you are right. we didn't change this file intentionally. Thanks for your explanation, we will fix it.